components/Nav/NavListItem/NavListItem.js

```diff
@@ -46,6 +46,14 @@ export default class NavListItem extends Component {
 
     const hasSublinks = props.sublinks.length > 0;
 
+    function tabDown() {
```

Comment:
You defined this method as `tabDown` but didn't consume it anywhere.

Did you test to see if this works?